app/src/main/java/io/homeassistant/companion/android/settings/SettingsPresenterImpl.kt

```diff
@@ -59,6 +59,7 @@ class SettingsPresenterImpl @Inject constructor(
                 "connection_internal" -> (urlUseCase.getUrl(true) ?: "").toString()
                 "connection_external" -> (urlUseCase.getUrl(false) ?: "").toString()
                 "registration_name" -> integrationUseCase.getRegistration().deviceName
+                "session_timeout" -> integrationUseCase.getSessionTimeOut().toString()
```

Comment:
Is this still needed?

Reply:
Yes, it's still necessary if the user is allowed to choose time in second to session timedout

Reply:
What exactly are we allowing to configure, how long the app can be unused until we ask again for biometrics ?

Reply:
Yes that's exactly it

Reply:
I'm personally a big believer in sane defaults that one cannot override. Having a 1000 options is going to be a pain to maintain. How will we show all the app options to the user while not hiding options?